docker/install

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+set -e
+
+# Install AFL
+wget http://lcamtuf.coredump.cx/afl/releases/afl-latest.tgz \
+    && tar -xzvf afl-latest.tgz \
+    && rm -rf afl-latest.tgz \
+    && cd afl-2.52b/ \
+    && make \
+    && sudo make install
+
+# Install Honggfuzz
+git clone https://github.com/google/honggfuzz \
+    && cd honggfuzz \
+    && make \
+    && sudo make install
+
+# Install Eclipser
+git clone https://github.com/SoftSec-KAIST/Eclipser \
+    && cd Eclipser \
+    && make
+
+# Install Angora
+git clone https://github.com/AngoraFuzzer/Angora \
+    && cd Angora \
+    && ./build/build.sh
+
+# Build and install DeepState under a variety of compilers
+cd deepstate \
+    && rm -Rf CMakeFiles CMakeCache.txt \
+    && rm -Rf build \
+    && mkdir -p build \
+    && cd build \
+
+    && rm -rf CMakeFiles CMakeCache.txt \
+    && CXX=clang++ CC=clang cmake -DDEEPSTATE_LIBFUZZER=ON ../ \
+    && sudo make install \
+
+    && rm -rf CMakeFiles CMakeCache.txt \
+    && CXX=afl-clang++ CC=afl-clang cmake -DDEEPSTATE_AFL=ON ../ \
+    && sudo make install \
+
+    && rm -rf CMakeFiles CMakeCache.txt \
+    && CXX=hfuzz-clang++ CC=hfuzz-clang cmake -DDEEPSTATE_HFUZZ=ON ../ \
+    && sudo make install \
+
+    && rm -rf CMakeFiles CMakeCache.txt \
+    && USE_TRACK=1 CXX=$ANGORA/bin/angora-clang++ CC=$ANGORA/bin/angora-clang cmake -DDEEPSTATE_ANGORA=ON ../ \
+    && export USE_TRACK=1 && sudo -E sh -c 'make -i install' \
+    && make clean \
+    && rm -rf CMakeFiles CMakeCache.txt \
+    && CXX=$ANGORA/bin/angora-clang++ CC=$ANGORA/bin/angora-clang cmake -DDEEPSTATE_ANGORA=ON ../ \
```

Comment:
`export USE_FAST=1` missing at the beginning of the command?
It will build library with `_taint.a` suffix